src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,177 @@
+package duke;
+
+import java.util.List;
+import java.util.Scanner;
+
+import duke.resource.TaskList;
+import duke.task.Task;
+import duke.util.DukeException;
+
+/**
+ * Ui class that handles the user interaction component of the bot.
+ */
+
+public class Ui {
+
+    private static final String LINE =
+            "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+
+    private final Scanner sc;
+
+
+    /**
+     * Constructor that creates a Ui object, with a Scanner that reads user input.
+     */
+
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Prints to user if the given save file has loaded successfully and the tasks found
+     * in the save file.
+     * @param tasks a TaskList containing the tasks in the save file
+     */
+
+    public void loaded(TaskList tasks) {
```

Comment:
Is it better if the loaded method is named as printLoaded instead to be more meaningful? :)

Reply:
I agree, thanks!